app/src/main/res/layout/movie_details_fragment.xml

```diff
@@ -14,6 +14,22 @@
         android:layout_height="match_parent"
         tools:context=".app.movie.details.view.MovieDetailsFragment">
 
+        <androidx.constraintlayout.widget.Group
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:isVisible="@{viewModel.isScreenElementsVisible}"
+            app:constraint_referenced_ids="scrollViewForDetails" />
```

Comment:
Since you have a one container for all elements (ScrollView) and you group references only one id then you can drop it and just handle visibility directly on ScrollView

Reply:
Yes, it works correctly and in this way, I can avoid redundant code :D. I add `app:isVisible="@{viewMode.isScreenElementVisible}"` to scrollView